### ISSUE TITLE
Marking a goal as achieved

### ISSUE THREAD

**Issue by jkbdbrnwsk** [x-team/unleash#463]
Clicking on the little circle on a timeline next to a goal triggers a popup.

The popup should appear with a few options:
- checkbox: send a notification to Slack
- input: additional comment to be attached to the notification
- button: Mark as achieved

After hitting "Mark as achieved", the circle gets filled with a grey checkmark.

![image](https://cloud.githubusercontent.com/assets/7212882/26693791/b191de08-4705-11e7-8a3f-ae7aca82d78b.png)

**Comment by jacekelgda** [comment]
I strongly disagree with using that dot for functional button. It makes little sense Imo. More natural would be to use `...` for functions such as edit, mark done etc. What do you think ?

**Comment by jkbdbrnwsk** [comment]
Yep, I like your idea better and I strongly agree. ;) 👍
